### ISSUE TITLE
CooCox IDE is discussed about in manual, but no project file in demo to use it

### ISSUE THREAD

**Issue by kalon33** [RAKWireless/RAK5205-WisTrio-LoRa#2]
Contrary to what I was used to with RAK811 trackerboard, there's no demo coIDE project, whereas it is listed in manual.

**Comment by RAKWireless** [comment]
I'm sorry, the new firmware don't support CoIDE because CoIDE will not maintain by official. You can use Keil now.

**Comment by kalon33** [comment]
Sure, but Keil is not free... 

----- Mail original -----

De: "RAKwireless" <notifications@github.com> 
À: "RAKWireless/RAK5205-WisTrio-LoRa" <RAK5205-WisTrio-LoRa@noreply.github.com> 
Cc: "Nicolas Derive" <kalon33@ubuntu.com>, "Author" <author@noreply.github.com> 
Envoyé: Lundi 7 Janvier 2019 15:51:24 
Objet: Re: [RAKWireless/RAK5205-WisTrio-LoRa] CooCox IDE is discussed about in manual, but no project file in demo to use it (#2) 



I'm sorry, the new firmware don't support CoIDE because CoIDE will not maintain by official. You can use Keil now.

**Comment by kastayou** [comment]
> I'm sorry, the new firmware don't support CoIDE because CoIDE will not maintain by official. You can use Keil now.

This is not inline with the documentation and I agree with Kalon33, Keil is not free.
Could you provide a project on an open and multiplatform IDE (eclipse, platformio)

**Comment by nufinuf** [comment]
> I'm sorry, the new firmware don't support CoIDE because CoIDE will not maintain by official. You can use Keil now.

The reason, why I bought it, was the option developing in CoIDE. Why you not support it. And if documentation mentioned support for CoIDE, is it that reason to get money back?

**Comment by afremont** [comment]
I'm guessing that the apparently dead product CooCox is not able to build the current version of the software or that there is some other valid reason for RAK dropping it, such as no one being at the other end to add support for the device or that RAK is tired of fielding complaints for a product they didn't produce and couldn't support if they wanted.  Honestly, I'm kinda surprised that many people still seem to be using it after all this time.  

Maybe it's time to try something else with a little more active support, such as Eclipse or Platformio, though it looks like Platformio may be headed down the same path as CooCox.  From what I've seen, it appears that CooCox was based on Eclipse, so.....  The 5205 is still a nice piece of hardware, So far it appears to be the best in class when it comes to low power usage.

**Comment by michaelshmitty** [comment]
I agree that the RAK5205 is a nice piece of hardware. Unfortunately the software quality of the firmware is not on the same level.
It would be great if this firmware was rewritten using and supporting the native ST toolset, which is free to use (but requires registration): [STM32CubeMX](https://www.st.com/content/st_com/en/products/development-tools/software-development-tools/stm32-software-development-tools/stm32-configurators-and-code-generators/stm32cubemx.html) for initial boilerplate code generation and [System Workbench for STM32](http://www.openstm32.org/System%2BWorkbench%2Bfor%2BSTM32) as IDE.

**Comment by Phando** [comment]
Have there been any updates on this? IS there a sample project for the RAK5205 for CooCox or for PlatformIO?
